src/components/footer.css

```diff
@@ -0,0 +1,52 @@
+/* footer */
+.footer {
+  background-color: var(--black-000);
+  display: flex;
+  justify-content: space-around;
+  padding: var(--sp-md);
+}
+
+.footer-columns {
+  display: flex;
+  flex-direction: column;
+  margin: 0px 12%;
```

Comment:
You need not specify 0px just 0 is enough and also using same unit is better like only % or px